gnucash/gtkbuilder/dialog-import.glade

```diff
@@ -14,6 +14,9 @@
     <property name="default_height">600</property>
     <property name="type_hint">dialog</property>
     <signal name="map" handler="gnc_ui_generic_account_picker_map_cb" swapped="no"/>
+    <child>
```

Comment:
I have in the passed removed them in a text editor if it was unrelated to a change but I see you are using a newer glade version, need to get that version and see.

Reply:
This is actually an artifact of a difference between master and maint, totally unrelated to Martin's work. It's from https://github.com/Gnucash/gnucash/commit/cae8ecde8f9ddd615ff47c783ce8136395bd71a9, part of PR #405. It seems to have gotten applied to master instead of maint and I need to figure out why and probably get it into maint.

Reply:
I believe PR #405 went on master as it implements a feature request and at the time of that PR I was still advocating to have new features on master instead of maint. There were too many moving targets in our effort to stabilize the 3.x branch to add random new features. I think I'm ready to relax this a bit as 3.x is now in a better shape than it was back then.

Reply:
OK, but not this time.
After working through separating the two PRs I looked at Martin's changes and decided that while it's big it would be too much code-copying to make a separate directory so I'm hand-applying the changes with ifdefs after having cleaned out all of the old pre AQBanking 5.3 ifdefs. That's the version in ubuntu 14.03LTS so there's no reason to drag around the older code.

The whole thing needs a refactor, there's a lot of code duplication, but there isn't time for that this week. I don't think there's really time for a proper code review either so I'm just going to get it building both ways and jam it in. More technical debt, sigh.